v4-tests/test_uxarray_fieldset.py

```diff
@@ -0,0 +1,64 @@
+
+import uxarray as ux
+from datetime import timedelta
+from parcels import (
+    FieldSet,
+    ParticleSet,
+    Particle,
+    UXPiecewiseConstantFace,
+    UXPiecewiseLinearNode,
+)
+import os
+
+# Get path of this script
+V4_TEST_DATA = f"{os.path.dirname(__file__)}/test_data"
```

Comment:
How will we be handling test data in v4 of Parcels? At the moment we download it via `parcels/tools/exampledata_utils.py`, but perhaps it would be better to:

- Generate the data (i.e., xarray datasets) on the fly in the idealised cases
- Download the data (from some hosting provider) for realistic cases. Perhaps using [`pooch`](https://pypi.org/project/pooch/) rather than our current custom downloading mechanism.

This will also mean that we can remove `parcels/data` (and any other committed data files) which I think would be good.

Let's continue this discussion in an issue :)

Reply:
#1950

Reply:
This is a good question. I liked @willirath 's suggestion when we were all together of using data posted to zenodo for the tests. Some of the tests (e.g. Stommel) can be generated on-the-fly with some helper functions. Removing data (nc files, etc) would be awesome; lighten things up.

Reply:
We'll need to update the `tests/v4` to use the FESOM channel data that was just merged in at https://github.com/OceanParcels/parcels-data/pull/1

Reply:
Resolved in https://github.com/OceanParcels/Parcels/pull/1946/commits/4bf5096b64674c3249dee5b26b5e7b1570555d06